framework/src/modules/chain/block_processor_v2.js

```diff
@@ -232,7 +232,18 @@ class BlockProcessorV2 extends BaseBlockProcessor {
 			data => this.blocksModule.verify(data),
 			data => this.blocksModule.apply(data),
 			({ block, tx }) => this.dposModule.apply(block, { tx }),
-			({ block, tx }) => this.bftModule.addNewBlock(block, tx),
+			({ block, tx }) =>
+				this.dposModule.getDelegateMinHeightActive(block, { tx }),
+			({ block, tx }, delegateMinHeightActive) => {
+				const blockHeader = {
+					...block,
+					// This parameter injected to block object to avoid big refactoring
+					// for the moment. `activeSinceRound` will be removed from the block
+					// object with https://github.com/LiskHQ/lisk-sdk/issues/4413
+					activeSinceRound: this.slots.calcRound(delegateMinHeightActive),
```

Comment:
In BFT module, this property is used to calculate the `delegateMinHeightActive` so I would suggest sending it as `delegateMinHeightActive` directly and skip the conversion to round.